test/functionalTest/cases/1492_using_limit=0_and_options=count_to_get_only_the_count_of_elements/using_limit=0_and_options=count_to_get_only_the_count_of_elements.test

```diff
@@ -0,0 +1,79 @@
+# Copyright 2021 Telefonica Investigacion y Desarrollo, S.A.U
```

Comment:
New name suggestion (to avoid using = character in the filename and make it much simpler):

test/functionalTest/cases/1492_limit_zero/limit_zero_entities.test

Reply:
In addition, I'd suggest to do a couple more tests, to cover subscriptions and registrations pagingation:

test/functionalTest/cases/1492_limit_zero/limit_zero_subs.test
test/functionalTest/cases/1492_limit_zero/limit_zero_regs.test

Same steps that limit_zero_entities.test (see the steps suggested here: https://github.com/telefonicaid/fiware-orion/pull/3994#discussion_r748310446) but using subs and regs instead of subs.

Reply:
Added in https://github.com/telefonicaid/fiware-orion/pull/3994/commits/8879a3975cdd73efa2a2d9289e528d99c68ff168